### PR TITLE
Add dev command line option

### PR DESCRIPTION
**Pull request by MartinHjelmare**
* Move argument parsing to options.py.
* Add dev option that starts the engine without modifying the env or updating the app.

### CODE REVIEW

File: imjoy/__main__.py [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+import json
 import os
 import subprocess
-import json
+import sys
+
+from imjoy.setup import parse_cmd_line
 
 
 def main():
```
Comment (oeway) [inline imjoy/__main__.py]: How about rename this to other names such as `start`, to avoid the same as the one we are importing from imjoyPluginEngine.

Comment (MartinHjelmare) [reply imjoy/__main__.py]: Yes, but I'd prefer to do that in another PR. There  are lots of lint issues left to fix. My plan is to go over one module at a time, and or one lint issue type at a time, after the refactor PR is merged.

Comment (oeway) [reply imjoy/__main__.py]: Ok, go ahead.

**Comment by oeway** [comment]
BTW, the name `setup.py` looks confusing when you think there is another one outside `imjoy` folder.

How about rename it to `options.py` or `args.py`?

**Comment by oeway** [comment]
This looks good to me, please merge.

And it should be easy to rebase the other PR.

**Pull request merged**